scss/main.scss

```diff
@@ -1133,7 +1140,20 @@ p.cardText {
   }
 }
 
-.portfolio-image-container {
+.gallery-2x4-grid {
```

Comment:
Instead of creating a new CSS selector, can we group all of these styles under "gallery-container"?

Note: We won't necessarily have 8 items here.

Reply:
I wasn't sure how to deal with needing two different grid-template-columns values in the gallery-container selector; the portfolio-gallery.html needs 3 columns and join-gallery.html only needs 2 columns. But everything else in gallery-container is needed for both pages. If I add the gallery-2x4-grid styles to gallery-container then portfolio-gallery.html will only have 2 columns instead of 3, right?

Reply:
Got it. I didn't know it was being used elsewhere.
Can we just change the `gallery-2x4-grid` name to something a little more generic? (Considering this is only one column on mobile, and we may not have 8 items)